### PR TITLE
Timesketch summit 2021

### PR DESCRIPTION
**Pull request by jaegeral**

### CODE REVIEW

File: docs/events/2021_timesketch_summit.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+your contribution to the summit. The call for presentations and workshops will
+run till **Feb 25th**.
+
+If you have ideas outside of the suggested formats, feel free to reach out.
+
+## Questions
+
+If you have any questions, please reach out to the team via 
+[slack in the Timesketch channel](https://github.com/open-source-dfir/slack) or
+via [Twitter](https://twitter.com/alexanderjaeger)
```
Comment (berggren) [inline docs/events/2021_timesketch_summit.md:53]: Let's skip personal Twitter accounts here so we don't spread the comms too much. Twitter will be the main outreach anyways and folks who wanna ask stuff there can just do it on the thread. :) WDYT?

I opt to use the Slack channel as the main communication channel, that way we also get the interested folks signed up and ready for the summit.

Please add the link to the self registration: http://join-open-source-dfir-slack.herokuapp.com/

**Review by berggren — CHANGES_REQUESTED** [review]

File: docs/events/2021_timesketch_summit.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Timesketch is an open source tool for collaborative forensic timeline analysis.
+Using sketches you and your collaborators can easily organize your timelines and
+analyze them all at the same time. Add meaning to your raw data with rich
+annotations, comments, tags and stars.
+
+## Agenda and talks
+
+The agenda will be posted on
+[Github](https://github.com/google/timesketch/tree/master/docs/events/2021_timesketch_summit.md)
+end of February.
```
Comment (kiddinn) [inline docs/events/2021_timesketch_summit.md:17]: add here... `before the end of February`

File: docs/events/2021_timesketch_summit.md (hunk 1 above)
Comment (kiddinn) [reply docs/events/2021_timesketch_summit.md:53]: agree

Comment (jaegeral) [reply docs/events/2021_timesketch_summit.md]: ACK, removed the twitter link, the self registration and description is in the linked github repo, that should be enough imho

**Review by berggren — APPROVED** [review]

**Pull request merged**